src/main/java/com/google/devtools/common/options/OptionsParser.java

```diff
@@ -699,7 +697,12 @@ public ImmutableList<String> parseWithSourceFunction(
     Preconditions.checkNotNull(priority);
     Preconditions.checkArgument(priority != OptionPriority.PriorityCategory.DEFAULT);
     OptionsParserImplResult optionsParserImplResult =
-        impl.parse(priority, sourceFunction, args, (OptionsData) fallbackData);
+        impl.parse(
+            priority,
+            sourceFunction,
+            Lists.transform(
```

Comment:
We have enough `Lists.transform` calls that it might make sense to have a method like:

```java
public static List<ArgAndFallbackData> wrapWithFallbackData(List<String> args, @Nullable OptionsData fallbackData)
```